lib/pqa/question_loader.rb

```diff
@@ -5,14 +5,16 @@ def initialize
       @client = ApiClient.new(uri, nil, nil, nil)
     end
 
-    def load_and_import(n = 1)
+    def load_and_import(n = 1, skip_import = false)
       import    = Import.new
       questions = (1..n).map do |i|
         QuestionBuilder.default("uin-#{i}")
       end
       load(questions)
-      import.run(Date.yesterday, Date.tomorrow)
-      questions
+      unless skip_import
+        import.run(Date.yesterday, Date.tomorrow)
+        questions
+      end
     end
```

Comment:
I find that is often better to create a new method rather than making the signature/internal logic of an existing one more complex. Perhaps we should have two different methods, and name them something like:

```ruby
def generate_and_import(n = 1)
   #...
end

def generate_and_api_load(n = 1)
   #...
end
```